src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 import spiceypy as spice
 
-from sorcha.ephemeris.simulation_constants import RADIUS_EARTH_KM
+from sorcha.ephemeris.simulation_constants import GMSUN, RADIUS_EARTH_KM
```

Comment:
This line is going to fail because GMSUN is not defined now with the two lines commented out in simulation_constants.py. Can we remove the import here for GMSUN